yt_dlp/extractor/jable.py

```diff
@@ -0,0 +1,125 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import re
+
+from .common import InfoExtractor
+from ..utils import (
+    InAdvancePagedList,
+    int_or_none,
+    orderedSet,
+    unified_strdate,
+)
+
+
+class JableIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?jable.tv/videos/(?P<id>[\w-]+)'
+    _TESTS = [{
+        'url': 'https://jable.tv/videos/pppd-812/',
+        'md5': 'f1537283a9bc073c31ff86ca35d9b2a6',
+        'info_dict': {
+            'id': 'pppd-812',
+            'ext': 'mp4',
+            'title': 'PPPD-812 只要表現好巨乳女教師吉根柚莉愛就獎勵學生們在白虎穴內射出精液',
+            'description': 'md5:5b6d4199a854f62c5e56e26ccad19967',
+            'thumbnail': r're:^https?://.*\.jpg$',
+            'age_limit': 18,
+            'like_count': int,
+            'view_count': int,
+            'upload_date': None,
+        },
+    }, {
+        'url': 'https://jable.tv/videos/apak-220/',
+        'md5': '71f9239d69ced58ab74a816908847cc1',
+        'info_dict': {
+            'id': 'apak-220',
+            'ext': 'mp4',
+            'title': 'md5:5c3861b7cf80112a6e2b70bccf170824',
+            'description': None,
+            'thumbnail': r're:^https?://.*\.jpg$',
+            'age_limit': 18,
+            'like_count': int,
+            'view_count': int,
+            'upload_date': '20220319',
+        },
+    }]
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+        webpage = self._download_webpage(url, video_id)
+        formats = self._extract_m3u8_formats(
+            self._search_regex(r'var\s+hlsUrl\s*=\s*\'([^\']+)', webpage, 'hls_url'), video_id, 'mp4', m3u8_id='hls')
+        self._sort_formats(formats)
+
+        return {
+            'id': video_id,
+            'title': self._og_search_title(webpage, default=None),
+            'description': self._og_search_description(webpage, default=None),
+            'thumbnail': self._og_search_thumbnail(webpage, default=None),
```

Comment:
`default=None` means you expect it to fail and it shouldn't show warning. That does make sense for thumbnail. But an empty description can be denoted by `''` as opposed to `None` (no description/can't extract description). For title, default=None makes no sense at all